pages/students/e22/e22452.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: studentDetails
+permalink: "/students/e22/452/"
+title: will be filled by GH action
```

Comment:
Why this
`title: will be filled by GH action`?

Reply:
<img width="1064" alt="Screenshot 2025-03-31 at 7 33 53 AM" src="https://github.com/user-attachments/assets/f1a619c9-680c-425d-a241-8fded1ab0bca" />

that is from the script `python_scripts add new student batch › akila_cs > & 1_gen_html.py›...`

Reply:
and it will be filled automatically by 
https://github.com/cepdnaclk/people.ce.pdn.ac.lk/blob/aa0a3374e210d1d7ec97b8e5536ccb8ad9808e3f/python_scripts/update_profile_from_google_form.py#L238

Reply:
I didn't understand what is being asking by ayya Can you explain whether Ayya is asking to run a Python script, or can I run Python scripts?

Reply:
@Thisen2002 Can you run the Python script mentioned by Ishan on your branch, and fill the `title` property correctly? Then we can merge the changes into main branch.

Reply:
@NuwanJ @Thisen2002 I think its easier to run the [1_gen_html.py](https://github.com/cepdnaclk/people.ce.pdn.ac.lk/blob/aa0a3374e210d1d7ec97b8e5536ccb8ad9808e3f/python_scripts/add_new_student_batch/akila_csv/1_gen_html.py#L27)

after doing the following the change
```suggestion
title: will be filled by GH action
title: {name}
```
rather than running the google docs reading  python script.

Reply:
Yeah, I can run that script.

Reply:
@NuwanJ I think @Thisen2002  has addressed all the concerns you mentioned.

can you please review again and merge the changes to the main branch.